themes/src/main/resources-community/theme/base/admin/messages/admin-messages_de.properties

```diff
@@ -706,16 +711,16 @@ group.assigned-roles.tooltip=Realm-Rollen die zur Gruppe zugeordnet sind
 #group.effective-roles-client.tooltip=Role mappings for this client. Some roles here might be inherited from a mapped composite role.
 
 group.move.success=Gruppe verschoben.
-group.remove.confirm.title=Gruppe l�schen
-group.remove.confirm.message=Sind Sie sicher, dass Sie die Gruppe \u201E{{name}}\u201C l�schen m�chten?
-group.remove.success=Die Gruppe wurde gel�scht.
+group.remove.confirm.title=Gruppe l�schen
+group.remove.confirm.message=Sind Sie sicher, dass Sie die Gruppe \u201E{{name}}\u201C l�schen m�chten?
+group.remove.success=Die Gruppe wurde gel�scht.
```

Comment:
Are changes like this intentional?

Reply:
Sorry it seemed to be an encoding problem on my side. Should be fixed now.